sbtplugin/src/main/scala/com/typesafe/tools/mima/plugin/SbtMima.scala

```diff
@@ -46,7 +46,9 @@ object SbtMima {
   /** Reports binary compatibility errors.
    *  @param failOnProblem if true, fails the build on binary compatibility errors.
    */
-  def reportErrors(problemsInFiles: Map[ModuleID, (List[core.Problem], List[core.Problem])],
+  def reportErrors(module: ModuleID,
```

Comment:
Yeah, it was probably a bit of an unnecessary, paranoid call. Fortunately the alternative change isn't too cumbersome. Thank you.